doc/release/RELEASE-NOTES.md

```diff
@@ -3,6 +3,19 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). The format is based on the `RELEASE-NOTES-TEMPLATE.md` file.
 
+## Release 1.32
+
+## Introduction
+* Product name: Open Supply Hub
+* Release date: March 22, 2025
+
+### Bugfix
+* [OSDEV-1806](https://opensupplyhub.atlassian.net/browse/OSDEV-1806) - Refactored the Parent Company field validation. The field is now validated as a regular character field.
+
+### Release instructions:
+* *Provide release instructions here.*
```

Comment:
Please copy/paste the regular release instructions.

* Ensure that the following commands are included in the `post_deployment` command:
    * `migrate`
    * `reindex_database`

Reply:
Done.